app/src/main/java/com/github/onlynotesswent/ui/user/NotificationScreen.kt

```diff
@@ -71,65 +74,90 @@ fun NotificationScreen(
         LazyColumn(
             contentPadding = innerPadding, modifier = Modifier.testTag("notificationsList")) {
               val sortedNotification = userNotifications.value.sortedByDescending { it.timestamp }
-              items(userNotifications.value.size) { index ->
-                // Display each notification
-                val notification = sortedNotification[index]
-                val senderUser: MutableState<User?> = remember { mutableStateOf(null) }
-
-                if (notification.senderId != null) {
-                  userViewModel.getUserById(
-                      notification.senderId, { user -> senderUser.value = user }, {}, {})
+              if (userNotifications.value.isEmpty()) {
+                item {
+                  Box(modifier = Modifier.fillMaxWidth()) {
```

Comment:
You could also add the loader from `ui.common` if you want, it takes already a custom text as parameter

Reply:
Thanks I will look into that

Reply:
Well we don't really need a loading screen because there is no notification, it's not loading so I will keep what I did before